networking/src/ApiClient.ts

```diff
@@ -80,6 +113,157 @@ interface SubscriptionValue<T> {
 }
 
 export class ApiClient implements IApiClient {
+
+    async createGameTemplate(
+        id: string,
+        title: string,
+        owner: string,
+        version: number,
+        description: string,
+        domain: string,
+        cluster: string,
+        grade: string,
+        standard: string,
+        phaseOneTime: number,
+        phaseTwoTime: number,
+        imageUrl: string
+    ): Promise<IGameTemplate | null> {
+        console.log("here");
+        const input: CreateGameTemplateInput = {
+           id,
+           title,
+           owner,
+           version,
+           description,
+           domain,
+           cluster,
+           grade,
+           standard,
+           phaseOneTime,
+           phaseTwoTime,
+           imageUrl
+        }
+        const variables: CreateGameTemplateMutationVariables = { input }
+        const gameTemplate = await this.callGraphQL<CreateGameTemplateMutation>(
+            createGameTemplate,
+            variables
+        )
+        if (
+            isNullOrUndefined(gameTemplate.data) ||
+            isNullOrUndefined(gameTemplate.data.createGameTemplate)
+        ) {
+            throw new Error(`Failed to create game template.`)
+        }
+        console.log(gameTemplate);
+        return null; //GameTemplateParser.gameTemplateFromAWSGameTemplate(gameTemplate.data.createGameTemplate) as IGameTemplate;
```

Comment:
intentional?

Reply:
this has been resolved in subsequent PRs